### ISSUE TITLE
Get log files failing: Is a directory

### ISSUE THREAD

**Issue by kinow** [BSC-ES/autosubmit#212]
In GitLab by @dmanubens on Nov 14, 2016, 11:46

```
15 of 19 jobs remaining (21:10)
^[[32mJob a011_19930501_fc1_1_ARPEGE_NEMO_SIM is COMPLETED^[[0m^[[39m
^[[31m[CRITICAL] Unhandled exception on Autosubmit: [Errno 21] Is a directory: '/home/Earth/dmanuben/debug/autosubmit/a011/tmp/LOG_a011/'
Traceback (most recent call last):
  File "/home/Earth/dmanuben/pyCharm/autosubmit/autosubmit/autosubmit.py", line 324, in parse_args
    return Autosubmit.run_experiment(args.expid)
  File "/home/Earth/dmanuben/pyCharm/autosubmit/autosubmit/autosubmit.py", line 659, in run_experiment
    if prev_status != job.update_status(platform.check_job(job.id)):
  File "/home/Earth/dmanuben/pyCharm/autosubmit/autosubmit/job/job.py", line 432, in update_status
    self.get_platform().get_logs_files(self.expid, self.out_filename, self.err_filename)
  File "/home/Earth/dmanuben/pyCharm/autosubmit/autosubmit/platforms/platform.py", line 176, in get_logs_files
    self.get_files([job_out_filename, job_err_filename], False, 'LOG_{0}'.format(exp_id))
  File "/home/Earth/dmanuben/pyCharm/autosubmit/autosubmit/platforms/platform.py", line 152, in get_files
    self.get_file(filename, must_exist, relative_path)
  File "/home/Earth/dmanuben/pyCharm/autosubmit/autosubmit/platforms/paramiko_platform.py", line 117, in get_file
    os.remove(local_path)
OSError: [Errno 21] Is a directory: '/home/Earth/dmanuben/debug/autosubmit/a011/tmp/LOG_a011/'
```

**Comment by kinow** [comment]
In GitLab by @dmanubens on Nov 14, 2016, 12:23

same problem here:
```
16 of 19 jobs remaining (21:47)
^[[33mJob a010_19930501_fc0_1_EC_EARTH_SIM is FAILED^[[0m^[[39m
^[[31m[CRITICAL] Unhandled exception on Autosubmit: [Errno 21] Is a directory: '/home/Earth/dmanuben/debug/autosubmit/a010/tmp/LOG_a010/'
Traceback (most recent call last):
  File "/home/Earth/dmanuben/pyCharm/autosubmit/autosubmit/autosubmit.py", line 324, in parse_args
    return Autosubmit.run_experiment(args.expid)
  File "/home/Earth/dmanuben/pyCharm/autosubmit/autosubmit/autosubmit.py", line 659, in run_experiment
    if prev_status != job.update_status(platform.check_job(job.id)):
  File "/home/Earth/dmanuben/pyCharm/autosubmit/autosubmit/job/job.py", line 432, in update_status
    self.get_platform().get_logs_files(self.expid, self.out_filename, self.err_filename)
  File "/home/Earth/dmanuben/pyCharm/autosubmit/autosubmit/platforms/platform.py", line 176, in get_logs_files
    self.get_files([job_out_filename, job_err_filename], False, 'LOG_{0}'.format(exp_id))
  File "/home/Earth/dmanuben/pyCharm/autosubmit/autosubmit/platforms/platform.py", line 152, in get_files
    self.get_file(filename, must_exist, relative_path)
  File "/home/Earth/dmanuben/pyCharm/autosubmit/autosubmit/platforms/paramiko_platform.py", line 117, in get_file
    os.remove(local_path)
OSError: [Errno 21] Is a directory: '/home/Earth/dmanuben/debug/autosubmit/a010/tmp/LOG_a010/'
```

**Comment by kinow** [comment]
In GitLab by @ jlope2 on Dec 23, 2016, 16:44

Hi @dmanubens,

I guess this error occurred while you were re-running (stop & re-run the process) an experiment.

This error was fixed here: 78c2850a04fc0d22874362029ab342086e6df359, so I proceed to close this issue.

Thanks!
